### PR TITLE
Add a new evm state dump command

### PR DESCRIPTION
**Pull request by pathornteng**
This PR adds a new evm state dump command which directly dump state out of evm.db
```
loom db evm-dump-3 --app-height 20  --storage-trie
```

- [ ] I added unit tests for any code that added
- [ ] I updated the CHANGELOG.md 
- [ ] All IP is original and not copied from another source
- [ ] I assign all copyright to Loom Network for the code in the pull request

### CODE REVIEW

File: cmd/loom/db/evm.go [hunk 1]
```diff
@@ -233,6 +237,105 @@ func newDumpEVMStateMultiWriterAppStoreCommand() *cobra.Command {
...
+			for it.Next() {
+				addrBytes := srcStateDBTrie.GetKey(it.Key)
+				addr := gcommon.BytesToAddress(addrBytes)
+				var data gstate.Account
+				if err := rlp.DecodeBytes(it.Value, &data); err != nil {
+					panic(err)
+				}
+
+				fmt.Printf("Account: %s\n", addr.Hex())
+				fmt.Printf("- Code: %x\n", srcState.GetCodeHash(addr))
```
Comment (enlight) [inline cmd/loom/db/evm.go]: Should dump the code too, not just the hash

File: cmd/loom/db/evm.go [hunk 2]
```diff
@@ -233,6 +237,105 @@ func newDumpEVMStateMultiWriterAppStoreCommand() *cobra.Command {
...
+
+				fmt.Println("---------------------------------------")
+			}
+			return nil
+		},
+	}
+
+	cmdFlags := cmd.Flags()
+	cmdFlags.Int64Var(&appHeight, "app-height", 0, "Dump EVM state as it was the specified app height")
+	cmdFlags.StringVar(&evmDBName, "evmdb-name", "evm", "Dump EVM state as it was the specified app height")
```
Comment (enlight) [inline cmd/loom/db/evm.go]: Copy pasta

File: cmd/loom/db/evm.go (hunk 1 above)
Comment (pathornteng) [reply cmd/loom/db/evm.go]: Fixed

File: cmd/loom/db/evm.go (hunk 2 above)
Comment (pathornteng) [reply cmd/loom/db/evm.go]: Fixed

**Pull request merged**